news/12608-ability-to-set-default-env

```diff
@@ -0,0 +1,21 @@
+### Enhancements
+
+* Ability to set a default conda environment at launch, so when you open an Anaconda Prompt, or enter `conda activate` at the command line, it will activate your preferred environment rather than `base`. 
+* The command to set the default conda environment is `conda config --set default_start_environment my-favourite-env`.
+* If you don't set a default start environment, `conda activate` will continue to activate `base`.
```

Comment:
```suggestion
```

It's better just have a single line per section. There's already plenty of information in the documentation, so no need to have so much here.